RecoLocalTracker/SiPixelRecHits/python/SiPixelRecHits_cfi.py

```diff
@@ -9,6 +9,11 @@
     VerboseLevel = cms.untracked.int32(0)
 )
 
+from Configuration.Eras.Modifier_phase2_brickedPixels_cff import phase2_brickedPixels
+phase2_brickedPixels.toModify(siPixelRecHits,
+                              CPE = 'PixelCPEGenericForBricked'
+)
```

Comment:
I'm confused how the modifiers are used to customize the `PixelCPEGenericESProducer` and the consumers. The ESProducer is customized with `phase2_tracker & phase2_3DPixels & phase2_brickedPixels & PixelCPEGenericForBricked` whereas here (and elsewhere) only either `phase2_brickedPixels` or `PixelCPEGenericForBricked` is used.

E.g. if `PixelCPEGenericForBricked` modifier is not enabled, the ESProducer customization is not applied, and consumers customized only with `phase2_brickedPixels` would fail to find the `PixelCPEGenericForBricked` ESProduct.

On the other hand, if the product label (`ComponentName`)  of `PixelCPEGenericESProducer` would not be changed, this customization would not be needed (as I mentioned above).

Reply:
This point has been discussed with @makortel and @mmusich.
There are pros and cons with both the approaches (e.g. keeping the code as it is or following the suggestion by Matti)
We keep the current implementation to be reviewed by the reco and offline managers